action_server/src/action_server/actions/navigate_to.py

```diff
@@ -94,11 +88,26 @@ def _configure(self, robot, config):
                 self._config_result.required_context['source-location'] = config.semantics['source-location']
             return
         # Now we can assume we know the navigation goal entity!
+        # operator
+        if semantics.target_location.id == 'operator':
+            # If we need to navigate to "me", which resolves to "operator", plant a waypoint at the current position to
+            # navigate to.
+            # TODO: learn to recognize the operator so that you know you found him later on
+            self._robot.ed.update_entity(uuid="operator", frame_stamped=self._robot.base.get_location(),
```

Comment:
Is this not done somewhere else in our software stack??

Reply:
Perhaps, but integration of that in the action server is not the goal of this PR. This is a bug fix of the current implementation